mdn/developer-portal
# Add nickname support to people pages

This changeset adds support for nicknames within people pages. If supplied, it'll display as "aka "____"" within the page's title (the person's full name).

## Key changes

- Add a "nickname" field to the person model.
- Add logic to conditionally include the "nickname" field within the page's title if supplied.

## How to test

- Add a nickname within a person's profile, and observe the changed page title.
- Ensure other title occurances (such as people cards, page title etc) aren't modified.
